manager/integration/tests/test_backing_image.py

```diff
@@ -421,3 +421,26 @@ def test_exporting_backing_image_from_volume():  # NOQA
         is the same as that of the 2nd volume.
     11. Do cleanup.
     """
+
+
+@pytest.mark.skip(reason="TODO") # NOQA
+@pytest.mark.backing_image  # NOQA
```

Comment:
Can you add these test cases in test_csi_snapshotter file instead of backing_image.